### PR TITLE
Improve running instructions

### PR DESCRIPTION
**Pull request by yoavram**
- two commands before Procfile as it's quicker and easier (although not as "good")
- remove --beat on windows
- wooey -> Wooey

### CODE REVIEW

File: docs/running.rst [hunk 1]
```diff
@@ -14,10 +14,23 @@ the site.
...
+
+You can run Wooey by calling two commands (you will need a
+separate process for each):
+
+::
+
+    python manage.py celery worker -c 1 --beat -l info
+    python manage.py runserver
+
+On Windows, remove the ``--beat`` option.
```
Comment (Chris7) [inline docs/running.rst]: Can you rephrase this to say under Windows the beat option may not be supported. If you encounter an error, omit this argument.  I'd rather handle cases where in the future it may work.

File: docs/running.rst [hunk 2]
```diff
@@ -33,14 +46,5 @@ Your server can then be run by the simple command:
...
-
-You can also run wooey by invoking two commands (you will need a
-separate process for each):
-
-::
-
-    python manage.py celery worker -c 1 --beat -l info
-    python manage.py runserver
+    
+On Windows, remove the ``--beat`` option.
```
Comment (Chris7) [inline docs/running.rst]: same caveat as mentioned above

File: docs/running.rst [hunk 3]
```diff
@@ -14,10 +14,23 @@ the site.
...
+    python manage.py celery worker -c 1 --beat -l info
+    python manage.py runserver
+
+On Windows, the ``--beat`` option might not be supported.
+
 Through a Procfile
 ------------------
 
-The simplest way to run Wooey is to use a Procfile with
+The best way to run Wooey is to use a Procfile with
```
Comment (Chris7) [inline docs/running.rst]: I'd like to keep the term simple. I guess it's from my science background, but I never say best for anything :). Also, a caveat to this method is you have buffered stdout, which may be non-intuitive to some users why the output is delayed.

File: docs/running.rst [hunk 4]
```diff
@@ -14,10 +14,23 @@ the site.
...
+
+You can run Wooey by calling two commands (you will need a
+separate process for each):
+
+::
+
+    python manage.py celery worker -c 1 --beat -l info
+    python manage.py runserver
+
+On Windows, the ``--beat`` option might not be supported.
```
Comment (Chris7) [inline docs/running.rst]: can you change might to may?

**Comment by Chris7** [comment]
Just 2 comments, after those I'm good to merge. Thanks for your help!

File: docs/running.rst (hunk 3 above)
Comment (yoavram) [inline docs/running.rst]: I agree about 'best', but I don't think that using a Procfile is the 'simplest' - the simplest is using two separate processes. How about 'recommended'?

File: docs/running.rst (hunk 4 above)
Comment (yoavram) [inline docs/running.rst]: 'on windows, the --best option may not be supported' sounds like it could never be supported, which is contrary to the previous comment you had on this line. What do you think?

**Comment by yoavram** [comment]
I've replied to your comments, let me know what you think.

File: docs/running.rst (hunk 3 above)
Comment (Chris7) [inline docs/running.rst]: +1 on recommended

File: docs/running.rst (hunk 4 above)
Comment (Chris7) [inline docs/running.rst]: may doesn't imply that. May means it's a possibility that it is not supported and accepts that at a future time, it may be.

**Comment by yoavram** [comment]
OK, fixed. Let me know if you'd like me to rebase this PR into a single commit.

**Comment by Chris7** [comment]
That'd be nice. Thanks @yoavram
